Update Azure Arc-enabled vSphere resource bridge scenario (#2203)
microsoft/azure_arc: docs/azure_arc_jumpstart/azure_arc_vsphere/resource_bridge/powershell/_index.md

>   az ad sp show --id 'ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7' --query '{name:displayName,objectId:objectId}'

Reviewer: ```suggestion
  az ad sp show --id 'ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7' --query '{name:displayName,objectId:id}'
```

  ```
  
    ![Screenshot of Azure resource provider for vSphere](./img/01.png)

### vCenter Prerequisites

> - VMware vSphere vCenter Server 8.

Reviewer: should we mention 7 or 8 since both are supported ?

Author: true, fixed

> - A virtual network that can provide internet access, directly or through a proxy. It must also be possible for VMs on this network to communicate with the vCenter server on TCP port (usually 443).

Reviewer 2: ```suggestion
- A virtual network that can provide internet access, directly or through a proxy. It must also be possible for VMs on this network to communicate with the vCenter server on a TCP port (usually 443).
```

- Change the environment variables according to your environment:
  - _`location`_ - the Azure region you want to deploy to
  - _`subscriptionId`_ - your Azure subscription ID
  - _`resourceGroupName`_ - the name of the Azure resource group you will create your resources in

>   - _`applianceName`_ - a name for the Azure Arc resource bridge appliance, this will the name that will be used in Azure.

Reviewer 2: Consider rephrasing the sentence

  - _`customLocationName`_ - the name of your Azure Arc custom location

>   - _`vcenterName`_ -  the name of your vCenter

Reviewer 3: ```suggestion
  - _`vcenterName`_ -  The name of your vCenter
```
maybe "The fully qualified domain name (FQDN) of your vCenter"?

  - _`vcenterFqdn`_ - your vCenter fully qualified name
  - _`vcenterUsername`_ - username to authenticate to vCenter
  - _`vcenterPassword`_ - password to authenticate to vCenter
  - _`spnClientId`_ - your service principal App ID
  - _`spnClientSecret`_ - your service principal secret
  - _`spnTenantId`_ - your Azure AD tenant ID
  - _`vmTemplate`_ - the Arc appliance template name
  - _`datacenter`_ - the vSphere datacenter where the appliance will be deployed. Make sure to include the full path, for example: '/Jumpstart - Datacenter'
  - _`datastore`_ - the vSphere datastore name where the appliance will be deployed. For example: 'NFS'
  - _`folder`_ - the vSphere folder where the template and appliance will be deployed. Make sure to include the full path, for example: '/Jumpstart - Datacenter/VMs'
  - _`dnsServer`_ - IP of the DNS server to be used for the appliance
  - _`gateway`_ - Gateway IP address to be used for the appliance
  - _`ipAddressPrefix`_ - Network IP address range in CIDR notation to be used by the appliance

>   - _`k8sNodeIpPoolStart`_ - IP range start for the IPs to be used by the appliance

Reviewer 2: ```suggestion
  - _`k8sNodeIpPoolStart`_ - Starting IP address from the subnet to be used by the appliance for node pool
```

>   - _`k8sNodeIpPoolEnd`_ - IP range end for the IPs to be used by the appliance

Reviewer 2: ```suggestion
  - _`k8sNodeIpPoolEnd`_ - End IP address from subnet to be used by the appliance for node pool
```

>   - _`datacenter`_ - the vSphere datacenter where the appliance will be deployed. Make sure to include the full path, for example: '/Jumpstart - Datacenter'
>   - _`datastore`_ - the vSphere datastore name where the appliance will be deployed. For example: 'NFS'
>   - _`folder`_ - the vSphere folder where the template and appliance will be deployed. Make sure to include the full path, for example: '/Jumpstart - Datacenter/VMs'
>   - _`dnsServer`_ - IP of the DNS server to be used for the appliance
>   - _`gateway`_ - Gateway IP address to be used for the appliance
>   - _`ipAddressPrefix`_ - Network IP address range in CIDR notation to be used by the appliance
>   - _`k8sNodeIpPoolStart`_ - Starting IP address from the subnet to be used by the appliance for node pool
>   - _`k8sNodeIpPoolEnd`_ - End IP address from the subnet to be used by the appliance for node pool
>   - _`segment`_ - Name of the virtual network or segment to which the appliance VM must be connected. Use the same name as it is displayed in your vCenter, for example: 'Jumpstart VM segment - (VLAN 111)'
>   - _`resourcePool`_ - the name of the vSphere resource pool to be used by the appliance. Make sure to include the full path, for example: '/Jumpstart - Datacenter/host/Jumpstart Cluster/Resources'

Reviewer 3: ```suggestion
  - _`datacenter`_ - The vSphere datacenter where the appliance will be deployed. Make sure to include the full path, for example: '/Jumpstart - Datacenter'
  - _`datastore`_ - The vSphere datastore name where the appliance will be deployed. For example: 'NFS'
  - _`folder`_ - the vSphere folder where the template and appliance will be deployed. Make sure to include the full path, for example: '/Jumpstart - Datacenter/VMs'
  - _`dnsServer`_ - IP of the DNS server to be used for the appliance
  - _`gateway`_ - Gateway IP address to be used for the appliance
  - _`ipAddressPrefix`_ - Network IP address range in CIDR notation to be used by the appliance
  - _`k8sNodeIpPoolStart`_ - Starting IP address from the subnet to be used by the appliance for the node pool
  - _`k8sNodeIpPoolEnd`_ - End IP address from the subnet to be used by the appliance for the node pool
  - _`segment`_ - Name of the virtual network or segment to which the appliance VM must be connected. Use the same name as it is displayed in your vCenter, for example: 'Jumpstart VM segment - (VLAN 111)'
  - _`resourcePool`_ - The name of the vSphere resource pool to be used by the appliance. Make sure to include the full path, for example: '/Jumpstart - Datacenter/host/Jumpstart Cluster/Resources'
```
